cli/cli_connect.go

```diff
@@ -18,23 +18,20 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
-// ConnectUsageText is shown next to account command by nordvpn --help
-const ConnectUsageText = "Connects you to VPN"
-
-// ConnectFlagGroupUsageText is shown next to group flag by nordvpn connect --help
-const ConnectFlagGroupUsageText = "Specify a server group to connect to"
-
-// ConnectArgsUsageText is shown by nordvpn connect --help
-const ConnectArgsUsageText = `[<country>|<server>|<country_code>|<city>|<group>|<country> <city>]
```

Comment:
`ConnectArgsUsageText` was "abused" to insert description as well (because as I wrote the template didn't support an actual description). 
The output looks exactly the same after these changes.